### PR TITLE
blend: add wireframe test case

### PR DESCRIPTION
**Pull request by dspmeng**
add wireframe test case to blend, and add option -d to dump
NV12 output for debug

### CODE REVIEW

**Comment by dspmeng** [comment]
please merge after https://github.com/01org/libyami/pull/597.

File: examples/blend.cpp [hunk 1]
```diff
@@ -533,6 +603,29 @@ class Blend
         return true;
     }
 
+    bool write(SharedPtr<VideoFrame>& frame)
+    {
+        VASurfaceID surface = (VASurfaceID)frame->surface;
```
Comment (xuguangxin) [inline examples/blend.cpp]: is it ok to use mapSurfaceToImage do the dump?

Comment (xuguangxin) [inline examples/blend.cpp]: Hi @dspmeng 
any update on this?

Comment (dspmeng) [inline examples/blend.cpp]: done

**Comment by xuguangxin** [comment]
merged as 887b93edd6994edeff88460774753cbd48764859
thanks mengjia

**Pull request closed without merging**